test/unit/conditions/context.test.ts

```diff
@@ -114,7 +120,7 @@ describe('context parameters', () => {
     const conditionExpr = new ConditionExpression(condition);
     expect(conditionExpr.contextRequiresSigner()).toBe(true);
     expect(() => conditionExpr.buildContext(provider, {}, undefined)).toThrow(
-      `Condition contains ${USER_ADDRESS_PARAM} context variable and requires a signer to populate`
+      `Cannot use ${USER_ADDRESS_PARAM} as a parameter without a signer`
```

Comment:
Yes, I did that on purpose - I remembered updating this line previously and decided to pick this wording.

Reply:
How about?

```
Signer required to satisfy ${USER_ADDRESS_PARAM} context variable in condition.
```

Two thoughts here:
1. It's already a done deal - the condition is already present so it's not that the condition can't use `:userAddress`, it's that the context can't be built.
2. It may not be a "parameter" in the context of a function call, it could be part of the return value test. In general, our terminology here has been "context variables".